apps/codeforafrica/src/lib/api.ghost/index.js

```diff
@@ -0,0 +1,35 @@
+import { getAllPosts, getPost } from "@/codeforafrica/lib/api.ghost/posts";
+
+export async function getAllOpportunities() {
+  const allPosts = await getAllPosts();
+  return allPosts.filter(
+    (post) => post.primaryTag.name.toLowerCase() === "opportunities"
```

Comment:
Use `equalsIgnoreCase` instead of generating new string with `toLowerCase`/`toUpperCase`.
```suggestion
    (post) => equalsIgnoreCase(primaryTag.name, "opportunities")
```